### PR TITLE
Test data endpoint

### PR DESCRIPTION
**Pull request by ielis**
Setup endpoints for real-world `Cohort`, `Family`, and `Phenopacket` instances.

### CODE REVIEW

**Comment by ielis** [comment]
@belkassaby I added example `Phenopacket` endpoints to `api/v1/test/phenopacket/*`. The `Phenopacket`s are based on examples in [Phenopacket tools](https://github.com/phenopackets/phenopacket-tools/tree/main/phenotools-cli/src/main/java/org/phenopackets/phenotools/examples).

I'll add real-life examples of `Family` and `Cohort` tomorrow.

File: phenopacketlab-restapi/src/main/java/org/monarchinitiative/phenopacketlab/restapi/controller/test/PhenopacketController.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import org.phenopackets.schema.v2.Phenopacket;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping(value = "${api.version}/test/phenopacket")
+public class PhenopacketController {
+
+    @GetMapping(value = "/aml", headers = "Accept=application/json")
+    public Phenopacket aml() {
+        return Examples.Phenopackets.acuteMyeloidLeukemia();
```
Comment (belkassaby) [inline phenopacketlab-restapi/src/main/java/org/monarchinitiative/phenopacketlab/restapi/controller/test/PhenopacketController.java:13]: See ResponseEntity comment

**Review by belkassaby — CHANGES_REQUESTED** [review]
I think we should return the endpoint response wrapped in a ResponseEntity. For the endpoints where things haven't been implemented, we put HttpStatus.NOT_IMPLEMENTED and for all the other ones where the result object is returned, HttpStatus.OK.

**Comment by ielis** [comment]
Hello @belkassaby, I wrapped the responses in `ResponseEntity` and set the appropriate `HttpStatus`.

**Pull request merged**